test/__snapshots__/app.test.js.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`App Component Renders according to snapshot 1`] = `ShallowWrapper {}`;
```

Comment:
take a look back onto the tests the snapshots should return html like text here

Reply:
Hi Kevin I believe this problem should be resolved!